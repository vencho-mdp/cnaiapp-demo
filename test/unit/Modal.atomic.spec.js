import { mount } from '@vue/test-utils'
import Modal from '@/components/TheModal.vue'

describe('VTitle', () => {
  it('should render correctly', async () => {
    const el = mount(Modal, {
      slots: {
        default: '<h1>Content inside Modal</h1>'
      }
    })
    const text = el.find('h1')
    const html = el.html()
    expect(text.exists()).toBe(true)
    expect(html).toMatchSnapshot()
  })
})
