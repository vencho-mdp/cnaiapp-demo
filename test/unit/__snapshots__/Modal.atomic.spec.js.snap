// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VTitle should render correctly 1`] = `
"<div class=\\"w-full h-full px-2 flex flex-col justify-center items-center mb-8\\">
  <div class=\\"p-6 border border-gray-light rounded-md w-full md:max-w-md shadow-md\\">
    <h1>Content inside Modal</h1>
  </div>
</div>"
`;
